### PR TITLE
New Wellington Colours

### LINKED ISSUE #118: Wellington New Routes
Yay, the routes are just obtained from SQL, so just need colors. But we should also group them nicely like #116 into Congestion Free/Frequent/Whatever. Needs to be done soon!

## Wellington City
- 1 (has variants, how are we handling this in the DB)
- 53, 54, 55 Cancelled
- 10, 11 Cancelled
- 12, 12e
- 25, 43, 44, cancelled
- 17, 17e
- 8 cancelled
- 18e
- 19, 19e
- 47 cancelled
- 23, 23e
- 21 cancelled
- 46, 50 cancelled
- 9 cancelled
- 25
- 29, 29e
- 4, 22, 29 cancelled
- 30x, 31x, 32x
- 4, 30, 31 cancelled
- 33, 34
- 3s, 3w cancelled
- 35, 36
- 5, 6 cancelled
- 37
- 21 cancelled

## Hutt Valley
- 80
- 113

## Paraparaumu
- 264
- 281
- 291

## Wairarapa
- 205 cancelled (changed to 200)

## Porirua
- Bunch of School Buses Cancelled (Shouldn't need to do anything about this)
- 60
- 60e
- 210/211 Cancelled.

### PR DESCRIPTION
**Pull request by consindo**

### CODE REVIEW

**Comment by mjdavidson** [comment]
please use a better name

**Comment by consindo** [comment]
This fixes #118

**Review by mjdavidson — APPROVED** [review]
I'm happy with this.

File: server-worker/lines/nz-wlg.js [hunk 1]
```diff
@@ -178,47 +76,64 @@ const sortLines = (lines) => {
     }
     return parsedA - parsedB
   }
-  lines.forEach((i) => {
+  lines.forEach(i => {
```
Comment (mjdavidson) [inline server-worker/lines/nz-wlg.js]: please use a more descriptive iterator

**Review by mjdavidson — CHANGES_REQUESTED** [review]

File: server-worker/lines/nz-wlg.js (hunk 1 above)
Comment (mjdavidson) [reply server-worker/lines/nz-wlg.js]: something like ```lines.foreach(line => {``` would be preferable

File: server-worker/lines/nz-wlg.js [hunk 2]
```diff
@@ -15,210 +15,124 @@ const regionEnum = {
...
-        } else if (parseInt(record.route_short_name) >= 80 && parseInt(record.route_short_name) < 200) {
-          lineGroups[regionEnum.hutt].items.push(record.route_short_name)
-        } else if (parseInt(record.route_short_name) > 200) {
-          lineGroups[regionEnum.porirua].items.push(record.route_short_name)
+        lineOperators[record.route_short_name] = record.agency_id
+        if (record.route_type !== 3) {
+          lineGroups[regionEnum.cf].items.push(record.route_short_name)
         } else {
-          lineGroups[regionEnum.central].items.push(record.route_short_name)
+          if (record.route_short_name.slice(0, 1) === 'N') {
```
Comment (mjdavidson) [inline server-worker/lines/nz-wlg.js:102]: would like some line comments to know what you're trying to achieve by this slice

**Review by mjdavidson — CHANGES_REQUESTED** [review]

File: server-worker/lines/nz-wlg.js (hunk 2 above)
Comment (consindo) [reply server-worker/lines/nz-wlg.js:102]: Checks if the first character is N?

Comment (mjdavidson) [reply server-worker/lines/nz-wlg.js:102]: y tho

**Review by mjdavidson — APPROVED** [review]

**Pull request merged**